tools/tcpdrop.bt

```diff
@@ -10,15 +10,18 @@
  *
  * This provides information such as packet details, socket state, and kernel
  * stack trace for packets/segments that were dropped via tcp_drop().
```

Comment:
`tcp_drop()` is not called in the new version so the description should be updated.